gapis/api/vulkan/api/query_pool.api

```diff
@@ -110,12 +110,13 @@ cmd void vkDestroyQueryPool(
 @threadSafety("system")
 @indirect("VkDevice")
 @threadsafe
-@no_replay
+//@no_replay
```

Comment:
Can you remove the @no_replay annotation entirely please?

Reply:
done